SlevomatCodingStandard/Sniffs/Attributes/AttributesOrderSniff.php

```diff
@@ -76,6 +81,15 @@ public function process(File $phpcsFile, $attributeOpenerPointer): void
 
 		} while (true);
 
+		if ($this->orderAlphabetically) {
+			uasort($attributesGroups, static function (array $attributesGroup1, array $attributesGroup2): int {
```

Comment:
This is strange. You sort the attributes groups before counting the actual order.

Reply:
There should be one `if-else` to get `$expectedOrder` (and `$actualOrder` if needed). Not there is some duplicated code.

Reply:
> There should be one if-else to get $expectedOrder (and $actualOrder if needed). Not there is some duplicated code.

Sorry, I don't get what you mean.

Reply:
There's duplicate code like `if ($expectedOrder === $actualOrder) {`

Reply:
Thanks. Got it. Refactored.